src/main/java/pl/mimuw/zpp/quantumai/backendui/controller/GradeController.java

```diff
@@ -0,0 +1,39 @@
+package pl.mimuw.zpp.quantumai.backendui.controller;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.multipart.MultipartFile;
+import pl.mimuw.zpp.quantumai.backendui.model.Grade;
+import pl.mimuw.zpp.quantumai.backendui.model.Problem;
+import pl.mimuw.zpp.quantumai.backendui.service.GradeService;
+
+import static org.springframework.http.MediaType.MULTIPART_FORM_DATA_VALUE;
+import static org.springframework.http.MediaType.TEXT_PLAIN_VALUE;
+
+@RestController
+@RequestMapping("/grade")
+@RequiredArgsConstructor
+public class GradeController {
+    private final GradeService gradeService;
+
+    @PostMapping(value = "/generateRequest", consumes = {MULTIPART_FORM_DATA_VALUE, TEXT_PLAIN_VALUE})
+    public ResponseEntity<String> generateGradeRequest(
+            @RequestParam String graphId,
```

Comment:
Może dać tablicę Stringów w body? Bo wtedy nie będziemy musieli x razy przesyłać solution.

Reply:
to w dalszej kolejności